rtengine/iptransform.cc

```diff
@@ -87,6 +88,114 @@ float normn (float a, float b, int n)
     }
 }
 
+#ifdef __SSE2__
+inline void interpolateTransformCubic(rtengine::Imagefloat* src, int xs, int ys, float Dx, float Dy, float &r, float &g, float &b, float mul)
+{
+    constexpr float A = -0.85f;
+
+    // Vertical
+    const float t1Vert = A * (Dy - Dy * Dy);
```

Comment:
Don't think this makes a difference after optimization by the compiler, but there are still common subterms used more than once (like ' Dy * Dy' or 't1Vert * Dy').